### DISCUSSION TITLE
PWAsForFirefox 2.0.1

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#185] [Announcements]
## Fixed

* A bug in GitHub Actions configuration that prevented [v2.0.0](https://github.com/filips123/PWAsForFirefox/releases/tag/v2.0.0) from being published.

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/PWAsForFirefox/releases/tag/v2.0.1'>PWAsForFirefox 2.0.1</a>.</em>

**Comment by daddy32** [comment]
Hello,

`winget` seems to still be downloading version 1.4.2 as of now (and complaining about unknown version 2.0.1 when specified explicitly).

**Reply by filips123** [reply]
PR to add 2.0.1 to winget repo still hasn't been merged. Until it's merged, you can download MSI file from releases page.

**Reply by daddy32** [reply]
OK, thanks.
